src/spec/create_extension_spec.py

```diff
@@ -14,36 +14,364 @@ def main():
+                doc="Wavelength that the filter is designed to pass or reflect.",
+                dtype="float",
+            ),
+            NWBAttributeSpec(
+                name="bandwidth_in_nm",
+                doc="Width of the wavelength range that the filter allows to pass through or blocks.",
+                dtype="float",
+                shape=(2,),
+            ),
+            NWBAttributeSpec(
+                name="filter_type",
+                doc="Type of filter (e.g., 'Excitation', 'Emission', 'Bandpass', 'Longpass', 'Shortpass').",
+                dtype="text",
+            ),
+            NWBAttributeSpec(
+                name="model",
+                doc="Model of the optical filter.",
+                dtype="text",
+                required=False,
+            ),
+        ],
+    )
+
+    fiber_photometry_table = NWBGroupSpec(
+        neurodata_type_def="FiberPhotometryTable",
+        neurodata_type_inc="DynamicTable",
+        doc="Extends DynamicTable to hold metadata on the Fiber Photometry system.",
+        datasets=[
+            NWBDatasetSpec(
+                name="location",
+                doc="Location of fiber.",
+                dtype="text",
+                shape=(None,),
+                neurodata_type_inc="VectorData",
+            ),
+            NWBDatasetSpec(
+                name="coordinates",
+                doc="Fiber placement in stereotactic coordinates (AP, ML, DV) mm relative to Bregma.",
+                dtype="float",
+                shape=(None, 3),
+                neurodata_type_inc="VectorData",
+                quantity="?",
+                attributes=[NWBAttributeSpec(name="unit", doc="coordinates unit", value="millimeters", dtype="text")],
+            ),
+            NWBDatasetSpec(
+                name="indicator",
+                doc="Link to the indicator object.",
+                dtype=NWBRefSpec(target_type="Device", reftype="object"),
+                shape=(None,),
+                neurodata_type_inc="VectorData",
+            ),
+            NWBDatasetSpec(
+                name="notes",
+                doc="Description of system.",
+                dtype="text",
+                shape=(None,),
+                neurodata_type_inc="VectorData",
+                quantity="?",
+            ),
+            NWBDatasetSpec(
+                name="optical_fiber",
+                doc="Link to the optical fiber device.",
+                dtype=NWBRefSpec(target_type="Device", reftype="object"),
+                shape=(None,),
+                neurodata_type_inc="VectorData",
+            ),
+            NWBDatasetSpec(
+                name="excitation_source",
+                doc="Link to the excitation source device.",
+                dtype=NWBRefSpec(target_type="Device", reftype="object"),
+                shape=(None,),
+                neurodata_type_inc="VectorData",
+            ),
+            NWBDatasetSpec(
+                name="commandedvoltage_series",
```

Comment:
```suggestion
                name="commanded_voltage_series",
```
Or `commanded_voltage` like you have in the diagram.